src/ngraph_mark_for_clustering.cc

```diff
@@ -61,6 +61,18 @@ static Status NGraphPlacementRequested(Node* node, bool& placement_ok) {
   return Status::OK();
 }
 
+static Status CheckIfOutputNode(const Node* node,
+                                std::vector<string> skip_these_nodes,
+                                bool& skip_it) {
+  for (string& f : skip_these_nodes) {
+    if (node->name() == f) {
+      skip_it = true;
+      return Status::OK();
+    }
+  }
+  return Status::OK();
```

Comment:
May be move this check as part of NGraphPlacementRequested like in the Captured pass

Reply:
I will remove the unused NGraphPlacementRequested function in a separate PR, so we can leave this check separate.